### ISSUE TITLE
[bug]: Setting Cover Images Fails Without Error

### ISSUE THREAD

**Issue by GlitchWitch** [makeplane/plane#6083]
### Is there an existing issue for this?

- [X] I have searched the existing issues

While it is possible this is a duplicate of #5817, we are not experiencing the other issues noted by that user and can upload photos in all other areas of the application without issue.

### Current behavior

When attempting to set a cover image, such as on a project or your own profile, the image does not actually get set.

### Steps to reproduce

As an example, when saving my profile the following request body is sent.

```
[...omitted for brevity...],
"avatar_url":"/api/assets/v2/static/redacted/","cover_image_url":"https://images.unsplash.com/photo-1721332154373-17e78d19b4a4?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=M3w2NzgxMTB8MXwxfGFsbHwxfHx8fHx8fHwxNzMyMjM1NzYyfA&ixlib=rb-4.0.3&q=80&w=1080",
[...omitted for brevity...]}
```

However, the following response received shows that the cover_image_url along with other fields such as the "Role" are not actually saved by the application.

```
{
    [...omitted for brevity...]
    "cover_image": null,
    "cover_image_asset": null,
    [...omitted for brevity...]
}
```

The video below shows the issue. In the video I attempt to set a new cover image via upslash, I update the project, and then refresh the page. I repeat this again to show the error remains.

[Screencast from 2024-11-21 18-51-17.webm](https://github.com/user-attachments/assets/ded28182-c3ba-4fbc-96d5-338fd3b12263)

### Environment

Production

### Browser

Google Chrome

### Variant

Self-hosted

### Version

v1.4.0 with prime-cli v1.7.0 a18c053

### Additional Notes

This is a freshly installed environment setup less than 24 hours ago. The issue persisted before upgrading to Pro. `prime-cli healthcheck` shows all clear.
![image](https://github.com/user-attachments/assets/d084f353-593e-430e-924f-3ada707819a6)

**Comment by pushya22** [comment]
HI @GlitchWitch  this issue has already been resolved and released in our cloud version. The update will be available for the self-hosted version very soon.

**Comment by GlitchWitch** [comment]
Awesome, thanks @pushya22! I'll keep an eye out for the release of the fix and report back :)

**Comment by GlitchWitch** [comment]
I can confirm this is fixed as of [Release v1.5.0](https://plane.so/changelog/commercial?version=1.5.0)
